### ISSUE TITLE
imagemagick - convert command no longer works in latest version [Bug]

### ISSUE THREAD

**Issue by anwareset** [owl4ce/dotfiles#86]
**Describe the bug**
`convert` command is a ImageMagick member, since the latest version **`7.0.11.7-1`** it has a bug to create framed window from `.scripts/shot-seldraw` script.

**How to Reproduce**
Run `convert` as those `shot-seldraw` script.
```bash
convert /tmp/2021-04-15-190944_740x187_scrot.png \( +clone -background black -shadow 25x10+0+0 \) +swap -background none -layers merge +repage /tmp/2021-04-15-190944_740x187_scrot.png-shadow.png
```

**Screenshots**
![2021-04-15-194518_1418x59_scrot](https://user-images.githubusercontent.com/25749089/114871151-2f53d800-9e23-11eb-8546-3beb790a99c5.png)
![2021-04-15-194530_965x119_scrot](https://user-images.githubusercontent.com/25749089/114871163-337ff580-9e23-11eb-9a17-abfc7502e601.png)

**Additional context**
I don't think the root cause is from this dotfile script, but from ImageMagick.
Kindly check and confirm, i found similiar issue on ImageMagick official repository [github.com/ImageMagick/ImageMagick/issues/#3549](https://github.com/ImageMagick/ImageMagick/issues/3549#issue-858395594)

**Comment by owl4ce** [comment]
hmm

**Comment by owl4ce** [comment]
I'll try update to x.x.x.7, currently on x.x.x.6. 7 is masked in my Gentoo

**Comment by owl4ce** [comment]
> I'll try update to x.x.x.7, currently on x.x.x.6. 7 is masked in my Gentoo

yeah, thats issue is valid.

**Comment by anwareset** [comment]
> > I'll try update to x.x.x.7, currently on x.x.x.6. 7 is masked in my Gentoo
> 
> yeah, thats issue is valid. I'm going to downgrade back into `7.0.11.6`

Hi owl4ce

As this issue is confirmed, so I'm going to close it.
If we use Arch-based, downgrade the ImageMagick[0] package to <= **`7.0.11.6`** and ignore this specific package from update[1], just like [#3 ](https://github.com/owl4ce/dotfiles/issues/3#issuecomment-688324366) .

Refferences:
[0] https://wiki.archlinux.org/index.php/downgrading_packages#Automation
[1] https://wiki.archlinux.org/index.php/pacman#Skip_package_from_being_upgraded
[2] https://man.archlinux.org/man/pacman.conf.5

**Comment by owl4ce** [comment]
Thanks for reporting ^^

**Comment by sheeepdev** [comment]
Just wanted to post an issue about this, but seen this issue and downgrading imagemagick helped :D Before i could not even screenshot an area because it would say "Processing image" which would never process, thanks :D

**Comment by anwareset** [comment]
ImageMagick has fixed this issue since version `7.0.11.8-1`
